helper-cli/src/main/kotlin/commands/packagecuration/SetCommand.kt

```diff
@@ -58,10 +60,23 @@ class SetCommand : CliktCommand(
         .convert { it.absoluteFile.normalize() }
 
     override fun run() {
-        val provider = FilePackageCurationProvider(packageCurationsFile, packageCurationsDir)
+        val ortResultInput = readOrtResult(ortFile)
```

Comment:
Adding a new function and the dropping the other separately hides a bit the behavioral changes. 
Is there any particular reason for not just changing / refactoring the existing `replace...`  function to the `add...` function?